test/system/test_server_tls_ws.hpp

```diff
@@ -17,20 +19,19 @@
 
 #include <mqtt_server_cpp.hpp>
 #include "test_settings.hpp"
-#include "test_ctx_init.hpp"
 
 namespace mi = boost::multi_index;
 namespace as = boost::asio;
 
-class test_server_tls_ws : ctx_init {
+class test_server_tls_ws {
 public:
-    test_server_tls_ws(as::io_context& ioc, MQTT_NS::broker::broker_t& b)
+    test_server_tls_ws(as::io_context& ioc, boost::asio::ssl::context&& ctx, MQTT_NS::broker::broker_t& b, uint16_t port = broker_tls_ws_port)
         : server_(
             as::ip::tcp::endpoint(
                 as::ip::tcp::v4(),
-                broker_tls_ws_port
+                port
             ),
-            std::move(ctx),
+            std::forward<boost::asio::ssl::context>(ctx),
```

Comment:
```suggestion
            MQTT_NS::force_move(ctx),
```

ctx is rvalue-reference not forwarding reference in your code.